www-aps/qtchan/qtchan-0.95.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="A 4chan browser written with qt5"
+DESCRIPTION="A Qt5 based desktop browser for 4chan, features:
+* browse and post on 4chan
+* auto update threads and auto download original file
+* download all media as original filename
+* tree style tabs and save state (./session.txt)
+* open files in OS's default application
+* search and filter."
+HOMEPAGE="https://github.com/siavash119/qtchan"
+COPYRIGHT="2017 siavash119"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/siavash119/qtchan/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="b86fc331d9d91ad5d1fd15ac404e79e65084d1db58598861f95aa76917170721"
+SOURCE_FILENAME="qtchan-$portVersion.tar.gz"
+SOURCE_DIR="qtchan-$portVersion"
+PATCHES="qtchan-$portVersion.patchset"
+#ADDITIONAL_FILES="qtchan.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qtchan$secondaryArchSuffix = $portVersion
+	app:OtterBrowser$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:g++$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage qtchan$secondaryArchSuffix \
+	"$appsDir"/QtChan
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	qmake ..
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	mkdir -p "$appsDir"
+	install -t "$appsDir" qtchan
+
+	mv "$appsDir"/qtchan "$appsDir"/QtChan
+
+# TODO: needs an icon
```

Comment:
You could add a tiny `additional-files/qtchan.rdef.in` with something like
```

resource app_signature "application/x-vnd.qtchan";
resource app_flags B_MULTIPLE_LAUNCH;

resource app_version {
	major = @MAJOR@,
	middle = @MIDDLE@,
	minor = @MINOR@,
	variety = B_APPV_FINAL,
	internal = 0,
	short_info = "qtchan",
	long_info = "@LONG_INFO@"
};
```
and uncomment this whole section, with one extra line:
```
		-e "s|@LONG_INFO@|$SUMMARY|" \
```
This would make it slightly easier for another contributor to add an icon.
What do you think?